packages/@react-spectrum/s2/src/AlertDialog.tsx

```diff
@@ -129,7 +129,8 @@ export const AlertDialog = forwardRef(function AlertDialog(props: AlertDialogPro
                 onPress={() => chain(close(), onCancel())}
                 variant="secondary"
                 fillStyle="outline"
-                autoFocus={autoFocusButton === 'cancel'}>
+                autoFocus={autoFocusButton === 'cancel'}
+                data-testid="rsp-AlertDialog-cancelButton">
```

Comment:
wonder if we should version the namespace or anything in case we extend to other components where there may be more than one accessible at a time

Reply:
there can be multiple of the same test id if said component is just rendered multiple times in an app though right (maybe not for the AlertDialog but if we extend these to other components)?

Reply:
Right, say there's two InlineAlerts on the page, one v3 one S2, do we want them to have them the same or different? the more I sit on it, the more I'm ok with them being the same